CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.11)
-project(fdb_doc_layer VERSION 1.6.4 LANGUAGES C CXX)
+project(fdb_doc_layer VERSION 1.6.5 LANGUAGES C CXX)
```

Comment:
Do we have 1.6.5 schedule? Otherwise we can go to 1.7.0 directly.

Reply:
For now, not sure.